diff --git a/SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c b/SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c
index 5379cd9..8306927 100644
--- a/SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c
+++ b/SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c
@@ -780,6 +780,12 @@ Tcg2GetEventLog (
     *EventLogTruncated = mTcgDxeData.EventLogAreaStruct[Index].EventLogTruncated;
   }
 
+  DEBUG ((DEBUG_INFO, "T2GET: fmt %x started %d lasa %lx last %lx size %x\n",
+    EventLogFormat, mTcgDxeData.EventLogAreaStruct[Index].EventLogStarted,
+    mTcgDxeData.EventLogAreaStruct[Index].Lasa,
+    (UINT64)(UINTN)mTcgDxeData.EventLogAreaStruct[Index].LastEvent,
+    (UINT32)mTcgDxeData.EventLogAreaStruct[Index].EventLogSize));
+
   // Dump Event Log for debug purpose
   if ((EventLogLocation != NULL) && (EventLogLastEntry != NULL)) {
     DumpEventLog (EventLogFormat, *EventLogLocation, *EventLogLastEntry, mTcgDxeData.FinalEventsTable[Index]);
@@ -970,6 +976,11 @@ TcgDxeLogEvent (
   //
   EventLogAreaStruct = &mTcgDxeData.EventLogAreaStruct[Index];
 
+  DEBUG ((DEBUG_INFO, "T2LOG: fmt %x started %d trunc %d size %x add %x\n",
+    EventLogFormat, EventLogAreaStruct->EventLogStarted,
+    EventLogAreaStruct->EventLogTruncated,
+    (UINT32)EventLogAreaStruct->EventLogSize, NewEventHdrSize + NewEventSize));
+
   if (EventLogAreaStruct->EventLogTruncated) {
     return EFI_VOLUME_FULL;
   }
